bin/packages/post-css-config.js

```diff
@@ -1,64 +1,7 @@
+const themes = require( '../../packages/base-styles/themes' );
```

Comment:
It should read it from the new package:
```suggestion
const { adminColorSchemes } = require( '@wordpress/base-styles' );
```

Those are admin color schemes. See: https://developer.wordpress.org/reference/functions/register_admin_color_schemes/.

`@wordpress/base-styles` still needs to be added as a `devDependency` in the root `package.json` file.

Reply:
Good catch; updated in https://github.com/WordPress/gutenberg/pull/17883/commits/2efbef6cc6be4bf82e624e5d19980e082f0da76c and https://github.com/WordPress/gutenberg/pull/17883/commits/cd163070caa65c39012b56f155b2d9cd7aabf014